### ISSUE TITLE
vagrant up fails with permission denied on file .vagrant.d/tmp/hosts.local

### ISSUE THREAD

**Issue by maxoys45** [smdahlen/vagrant-hostmanager#175]
Windows 7
Vagrant 1.8.1
Vagrant Host Manager 1.7.0

Attempting to vagrant up for the first time on Windows 7 (works fine on my Mac) - fails with a permission denied error: http://pastebin.com/54aB76YM (Line 42 onwards)

Not sure if it's an issue with /etc/hosts or the hosts.local file. Apologies if this is not the normal way to post an issue, i'm just the front-end guy getting lost with vagrant :)

**Comment by seth-reeser** [comment]
The pastebin is no longer, can you please attach directly here, please?

**Comment by maxoys45** [comment]
Ah sorry about that, attached!
[vagrant.txt](https://github.com/smdahlen/vagrant-hostmanager/files/127098/vagrant.txt)

**Comment by seth-reeser** [comment]
Can you please see what the permissions of the C:/Users/Max/.vagrant.d/tmp/hosts.local file is?

**Comment by maxoys45** [comment]
Hey - I'd already set the permissions for everyone, admin, system to all for that file and all parent folders with no luck. thanks though!

**Comment by seth-reeser** [comment]
Try running `del /f C:/Users/Max/.vagrant.d/plugins.json` and `rmdir /s /q C:/Users/Max/.vagrant.d/gems`. Then re-install vagrant-hostmanager with an administrator command prompt.

**Comment by maxoys45** [comment]
nope, same error. agh this is driving me insane! thanks for continuing to try and help me :)

**Comment by soccerjustinh1** [comment]
I am getting this problem to with centos 6 host machine.
